tasks/install.yml

```diff
@@ -29,15 +29,25 @@
     owner: "{{ pyenv_owner }}"
     mode: "0644"
 
-- name: "Load pyenv env variables in {{ pyenv_setting_path }}"
-  ansible.builtin.lineinfile: dest="{{ pyenv_setting_path }}"
+- name: Use legacy variable pyenv_setting_path if available
+  ansible.builtin.set_fact:
+    pyenv_shellrc_file: "{{ pyenv_setting_path }}"
+  when: pyenv_setting_path is defined
+
+- name: Print deprecation warning if pyenv_setting_path defined
+  ansible.builtin.debug:
+    msg: pyenv_setting_path is deprecated, use pyenv_shellrc_file instead
+  when: pyenv_setting_path is defined
+
+- name: "Load pyenv env variables in {{ pyenv_shellrc_file }}"
+  ansible.builtin.lineinfile: dest="{{ pyenv_shellrc_file }}"
```

Comment:
Do you mind adding file permissions mode to this file? Ansible recommends explicits definition and in most cases I check what are the original permissions of the file.

Reply:
@staticdev This is not a row that I have  added,  only changed the name of the parameter. OK with any changes/improvements you would like to do in your code.

Reply:
@staticdev , looks like it would be quicker to approve this PR and then fix the issues in #161 . This PR does not introduce any new issues. The old ones are solved in #161 but that one needs github changes to make the automatic lint in github ehrn building the environment work.